src/charactersheet/viewmodels/common/notes/index.js

```diff
@@ -54,11 +54,21 @@ export function NotesViewModel() {
         self.selectNote(note);
     };
 
+    self.noteToSelect = function(note) {
```

Comment:
Minor nitpick, can this be called something like `selectPreviousNote`? That gives more insight as to what it does at a glance.

Reply:
lol I originally named the function selectPreviousNote, but I changed it because it doesn't always select the previous note. If the note deleted is the first note, it selects the note after the deleted note. I'm open to change it though.